scripts/generate_landings_risk_report.py

```diff
@@ -846,7 +846,11 @@ def go(self, days: int) -> None:
 
         meta_bugs = self.get_meta_bugs(days)
 
-        last_modified = db.last_modified(bugzilla.BUGS_DB)
+        try:
+            last_modified = db.last_modified(bugzilla.BUGS_DB)
+        except Exception:
+            raise
```

Comment:
No need to catch and re-raise, we can just rely on Python to propagate the exception.

Reply:
Ok

Reply:
Done